streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -4759,6 +4760,126 @@ public void shouldListNotPausedTasks() {
         assertEquals(taskManager.notPausedTasks().size(), 0);
     }
 
+    @Test
+    public void shouldRecyclePendingTasksFromStateDirectoryAsActive() {
+        final StandbyTask pendingTask = StreamsTestUtils.TaskBuilder.standbyTask(taskId00, taskId00ChangelogPartitions).build();
```

Comment:
nit: `standbyTask()` and `statefulTask()` are actually already statically imported into this test class. So could just use the following to make the code more readable: 
```suggestion
        final StandbyTask pendingTask = standbyTask(taskId00, taskId00ChangelogPartitions).build();
```

Reply:
Woops! Cleaned up in ada808f